### DISCUSSION TITLE
Is there a way to make the docker image not stop when using UltimateAutoRestart

### DISCUSSION THREAD

**Discussion by ChristopherHaws** [itzg/docker-minecraft-server#1171] [Q&A] [answered]
When I have the UltimateAutoRestart plugin running on my server and it triggers a restart, the docker image stops running. Is there a way to make this not happen?

Thanks!

**Comment by itzg** [comment, marked as answer]
I can't tell how it works but maybe configuring the container to auto-restart would help.

**Reply by ChristopherHaws** [reply]
Yup, that did it. I added the following flags to the docker command `--restart unless-stopped`. Thanks! :)
